cwel/component/button/button.es

```diff
@@ -0,0 +1,16 @@
+angular.module('cwel').directive('button', () => {
+    return {
+        restrict: 'C',
+        scope: {},
+        link(scope, el) {
+            console.log(el);
```

Comment:
@NathanRodger Can you remove this? Shouldn't our linting catch these @dstuessy? Maybe not during dev but our commit hook should catch them.

Reply:
You're right. It's been turned off for some reason. Gonna change this